test/test_git_helpers.py

```diff
@@ -0,0 +1,43 @@
+import time
+
+import conda_build_prepare.git_helpers as gh
+
+COMMIT_DELAY = 1
+
+
+def git_commit_and_tag(filepath, msg, tag=None):
+    now = time.time()
+    with open(filepath, 'w') as f:
+        f.write(f'{now}')
+    gh._call_custom_git_cmd(filepath.parent, f'add {filepath}')
+    gh._call_custom_git_cmd(filepath.parent, f'commit -m {msg}')
+    if tag:
+        gh._call_custom_git_cmd(filepath.parent, f'tag {tag}')
+    time.sleep(COMMIT_DELAY)
+
+
+def test_get_first_reachable_tag(tmp_path):
```

Comment:
I think you probably want to put the setup into a `def setup_test_repo` type function?

Reply:
you mean the `git init` + `git config`? Maybe we can do this when there is actually more than one test that share the same setup.

Reply:
Yeah, mostly because it wasn't clear to see what was the actual test verse what was just needed to get things set up for the test.